scripts/assemble_trace.py

```diff
@@ -217,6 +217,8 @@ def assemble_traces(args):
 
     stream = len(args.log) == 0
     for line in fileinput.input(args.log[0] if not stream else ("-",)):
+        if line[0] == '"':
```

Comment:
Is a `"` expected? Comments on why would be useful.

Reply:
It breaks the python JSON parser for some reason and it occurs on exactly 1 line in the import logs